stepup-build2.sh

```diff
@@ -103,7 +103,7 @@ fi
 PHP_VERSION=56
 INCLUDE_COMPOSER_BOOTSTRAP=no # Whether to include Symfony bootstrap cache and composer in the archive
 ENCORE=no # Whether to run yarn encore
-NODEJS_VERSION=10 # Nodejs version for building is 10, Symfony 3 gw, ss and ra use nodejs 6 during deploy
+NODEJS_VERSION=14 # Nodejs version for building is 14, This can be set in the component_info file 
```

Comment:
Do not modify this line. This sets the default node version. Changing this breaks building older components (i.e. any component that does not explicitly set a NODEJS_VERSION or that has no component_info). Set the required node version in the component_info of the component, that will override the default set here.